package.json

```diff
@@ -9,33 +9,30 @@
   "devDependencies": {
     "aws-sdk": "^2.300.0",
     "backstopjs": "^3.5.5",
-    "chokidar": "^2.0.4",
-    "concurrently": "^3.6.1",
     "fastly": "^2.2.0",
     "lerna": "^2.11.0",
     "map-stream": "0.0.7",
     "mime": "^2.3.1",
     "serve": "^9.6.0",
-    "vinyl-fs": "^3.0.3",
     "zlib": "^1.0.5"
   },
   "scripts": {
-    "build:styles": "lerna exec --scope @carto/airship-style -- npm run build",
+    "build:components": "lerna exec --scope @carto/airship-components npm run build",
```

Comment:
As far as I see `exec` command still requires `--` before the command

https://github.com/lerna/lerna/tree/master/commands/exec#readme

Reply:
From the doc you just pasted :trollface: 

> A double-dash (--) is necessary to pass dashed flags to the spawned command, but is not necessary when all the arguments are positional.

Reply:
Fair enough xDDD